library/std/src/sys/thread_local/os.rs

```diff
@@ -66,19 +66,19 @@ impl<T: 'static> Key<T> {
     /// The resulting pointer may not be used after reentrant inialialization
     /// or thread destruction has occurred.
     pub fn get(&'static self, i: Option<&mut Option<T>>, f: impl FnOnce() -> T) -> *const T {
-        // SAFETY: (FIXME: get should actually be safe)
-        let ptr = unsafe { self.os.get() as *mut Value<T> };
+        let key = self.key.force();
+        let ptr = unsafe { get(key) as *mut Value<T> };
```

Comment:
This & other instances in this file need safety comments. They didn't really have them previously, but we should keep the FIXME around if nothing else.

Reply:
I was the one to add the FIXME, but it's no longer right: the free `get` function relies on only receiving correct keys whereas the `get` method didn't even take an explicit key.

Sure, I'll add safety comments!

Reply:
Ah, okay! Even stating that is reasonable.